### PR TITLE
app-admin/rcm: Bump to version 1.3.4

### PR DESCRIPTION
**Pull request by fgtham**

### CODE REVIEW

File: app-admin/rcm/rcm-1.3.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Authors
```
Comment (hydrapolic) [inline app-admin/rcm/rcm-1.3.4.ebuild:1]: 2019 -> 2020 :)

File: app-admin/rcm/rcm-1.3.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (hydrapolic) [inline app-admin/rcm/rcm-1.3.4.ebuild:4]: Please switch to EAPI=7

**Comment by hydrapolic** [comment]
You need to add "Signed-off-by: Name <e-mail>" to your commit message.

File: app-admin/rcm/rcm-1.3.4.ebuild (hunk 1 above)
Comment (fgtham) [reply app-admin/rcm/rcm-1.3.4.ebuild]: fixed

File: app-admin/rcm/rcm-1.3.4.ebuild (hunk 2 above)
Comment (fgtham) [reply app-admin/rcm/rcm-1.3.4.ebuild]: fixed

**Comment by hydrapolic** [comment]
Please squash the 2 commits into 1.

**Comment by fgtham** [comment]
@hydrapolic is there anything else left to do?

**Comment by hydrapolic** [comment]
Looks good to me, but I cannot merge it.

File: app-admin/rcm/rcm-1.3.4.ebuild (hunk 1 above)
Comment (thesamesam) [reply app-admin/rcm/rcm-1.3.4.ebuild]: Use repoman to commit in future to avoid ;)

**Comment by thesamesam** [comment]
Thank you (@fgtham for the contribution and @hydrapolic for reviewing)!

**Pull request closed without merging**